src/mesh/mesh_triangle_interface.C

```diff
@@ -20,9 +20,7 @@
 
 #ifdef LIBMESH_HAVE_TRIANGLE
 
-// C/C++ includes
-#include <sstream>
-
+// llibmesh includes
```

Comment:
llibmesh?

Reply:
Nice catch, fixed in 87a981396903dca0148f4ca4d6b9b9abf2040805